spring-integration-file/src/main/java/org/springframework/integration/file/config/FileParserUtils.java

```diff
@@ -118,7 +124,7 @@ static void configureFilter(BeanDefinitionBuilder synchronizerBuilder, Element e
 			}
 			if (count != 1) {
 				parserContext.getReaderContext().error("at most one of 'filename-pattern', " +
-						"'filename-regex', 'filter' or 'filter-expression' is allowed on remote file inbound adapter",
+								"'filename-regex', 'filter' or 'filter-expression' is allowed on remote file inbound adapter",
```

Comment:
Thank you @snicoll for sharing your pain as  well 😄 

I think we with @garyrussell will work out some formal rules to follow with formatting.